docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/advanced-operations/snapshot-restore.md

```diff
@@ -15,63 +15,67 @@ Volume restore from an existing snapshot will create an exact replica of a stora
 To begin, you will need to create a StorageClass that defines the properties of the snapshot to be restored. Refer to [StorageClass Parameters](../../replicated-pv-mayastor/rs-configuration.md#storage-class-parameters) for more details. Use the following command to create the StorageClass:
 
 :::info
-thin: "true" and repl: "1" is the only supported combination.
+If you want to create a volume restore from snapshots, thin: "true" is required.
```

Comment:
For creation of volume restore from snapshots, storage class parameter, `thin: "true"` is required.

Reply:
I would add a bit more, something like this:

```suggestion
As it stands today, restores are thin-provisioned volumes created from a snapshot, and as such the storage class for the restored volume must also specify `thin: "true"`.
```